crates/viewer/re_ui/src/modal.rs

```diff
@@ -230,3 +272,34 @@ impl ModalWrapper {
         });
     }
 }
+
+/// Utility to produce a [`egui::Frame`] with padding on some sides.
+#[allow(clippy::fn_params_excessive_bools)]
+#[inline]
+fn view_padding_frame(left_and_right: bool, top: bool, bottom: bool) -> egui::Frame {
```

Comment:
I agree with clippy on this one, introducing a struct would make the code above much clearer.
```rust
struct ViewPaddingFrameParams {
    left_and_right: bool,
    top: bool,
    bottom: bool,
}
```

```rust
view_padding_frame(ViewPaddingFrameParams { left_and_right: false, top: false, bottom: true});
```